Makefile.am

```diff
@@ -90,3 +90,11 @@ test_test_scrypt_SOURCES =	test/test_scrypt.c			\
 			lib/crypto/crypto_scrypt_smix.c			\
 			cpusupport-config.h
 test_test_scrypt_LDADD=	libscrypt_sse2.a
+
+.PHONY: test
+# we can't only build "scrypt test/test_scrypt" because that won't build
+# the BUILT_SOURCES.
```

Comment:
If we don't have `BUILT_SOURCES`, then the compile will fail because it doesn't have `cpusupport.h`.  Remember, the way that autotools works in scrypt at the moment is that `make all` works fine, but (from an empty builddir), `make scrypt` will **fail**, because it doesn't build `cupsupport.h`.

scrypt depends on having proper crockery, after all!